src/components/ImportFile/ImportFile.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { ImportFile, ImportFileContent } from './index';
+
+const files = [];
+
+test('ImportFile is working properly', () => {
+  const component = shallow(
+    <ImportFile acceptedTypes="image/png" maxAmount={4} maxSize={4000} />
+  );
+  component.instance().onDrop(files, files);
+  component.instance().reset();
+  component.instance().onDropAccepted();
+  component.instance().onDropRejected();
+  component.instance().onDragEnter();
+
+  expect(component.render()).toMatchSnapshot();
+});
+
+test('ImportFileContent is working properly', () => {
```

Comment:
Targeted assertions would be better here.  this test at a glance does not indicate what "working properly" is.  You also might be able to get by with shallow here since refs are not coming into play.

Reply:
Needs to be moved to a different file as well. Testing 1 component per file makes it easier to find related tests and maintain in the future.

Reply:
Most of the project's component are creating 1 test file, just trying to follow the guidelines here. Is 1 per file new since using `enzyme`?

Reply:
Sorry, I wrote in another comment as well. Testing best guidelines are being revisited here: https://github.com/patternfly/patternfly-react/issues/287 this can remain as is.